src/app/content/components/AssignedTopBar.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled, { css } from 'styled-components';
+import theme from '../../theme';
+import { setTextSize } from '../actions';
+import * as selectContent from '../selectors';
+import { LinkedArchiveTreeSection } from '../types';
+import { TextResizer } from './Topbar/TextResizer';
+
+// tslint:disable-next-line:variable-name
+const TopBar = styled.div`
```

Comment:
would it be better to re-use the existing bar? isn't the design the same? we could at least use the `Styled.TopBarWrapper` component

Reply:
Yeah good call to use TopBarWrapper here, updated. I didn't want to use SearchInputWrapper/MobileSearchWrapper because of how much is specific to the complex layout on the main app, and they do a lot that I'm hoping we refactor at some point (maybe that's what we do now, extract these styles into a generic TopBar component?)

Reply:
Yea idk, I agree on avoiding all the complicated logic we don't need about the search results but if seems like there should be a shared component just for the border style and size

Reply:
Oops just saw this one after I made the other comments... This makes sense to me as well. It looks simple enough to make a wrapper component for the shadow/border/sticky/background styles, I'll spend a few extracting those.